ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -28,7 +28,7 @@
 from timeout_decorator.timeout_decorator import TimeoutError
 from ovs.extensions.generic.configuration import NotFoundException
 from ovs.extensions.generic.system import System
-from ovs.extensions.healthcheck.decorators import ExposeToCli
```

Comment:
General remark: You committed using CRLF io LF
On line 123 you have a potentially referenced before assignment issue